src/CSET/operators/aggregate.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2022 Met Office and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Operators to aggregate across either 1 or 2 dimensions."""
+
+import iris
+import iris.analysis
+import iris.coord_categorisation
+import iris.cube
+import isodate
+
+
+def aggregate(
```

Comment:
```suggestion
def time_aggregate(
```
Given this is now time-specific, it should be named as such. The workflow recipe will need updating too.